src/main/java/org/fundacionjala/coding/raul/movies/Rental.java

```diff
@@ -0,0 +1,73 @@
+package org.fundacionjala.coding.raul.movies;
+
+import java.util.logging.Logger;
+
+class Rental {
+    private Movie movie;
+    private int daysRented;
+    private static final double INCREMENT = 1.5;
+    private static final int THREE = 3;
+
+
+    Rental(final Movie movie, int daysRented) {
+        this.movie = movie;
+        this.daysRented = daysRented;
+    }
+
+    public int getDaysRented() {
+        return daysRented;
+    }
+
+    public Movie getMovie() {
+        return movie;
+    }
+    public double statement(final Movie movie) {
+        double thisAmount = movie.statement();
+        switch (movie.getPriceCode()) {
```

Comment:
Don't use the switch case.
In this case you can use polymorphism

Reply:
Done.